x-pack/platform/plugins/shared/triggers_actions_ui/public/application/sections/action_connector_form/action_type_form.test.tsx

```diff
@@ -381,7 +381,7 @@ describe('action_type_form', () => {
     jest.useFakeTimers({ legacyFakeTimers: true });
     wrapper.find('[data-test-subj="action-group-error-icon"]').first().simulate('mouseOver');
     // Run the timers so the EuiTooltip will be visible
-    jest.runAllTimers();
+    jest.runOnlyPendingTimers();
```

Comment:
fixing [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/267344#019469ff-7fb8-4380-9dd2-bb74307f8cf2) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/267344/jobs/019469ff-7fb8-4380-9dd2-bb74307f8cf2/artifacts/01946a25-21e3-410d-a0da-6a233469e5ee) Jest Tests #4 / ConnectorForm calls onChange when the form is invalid

I assumed that runAllTimers caused tooltip to close before we checked that it is present 🤷‍♂️